### DISCUSSION TITLE
valkey latest release support rhel 8.6 ?

### DISCUSSION THREAD

**Discussion by devops-subi** [valkey-io/valkey#1275] [Q&A]
I am getting this error:
/usr/local/bin/valkey-server: /lib64/libm.so.6: version `GLIBC_2.29' not found (required by /usr/local/bin/valkey-server)
/usr/local/bin/valkey-server: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by /usr/local/bin/valkey-server)
/usr/local/bin/valkey-server: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by /usr/local/bin/valkey-server)
/usr/local/bin/valkey-server: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by /usr/local/bin/valkey-server)

**Comment by avifenesh** [comment]
@devops-subi Did you manage since then?
Iv'e tested on RHEL 8.6 image, and CentOS 8.6 image and it works. Dockers though.
Needed to install `dnf install epel-release` and then `dnf install valkey`.
`RHEL` might have it OOB, I don't have an account, so I couldn't get to their repositories from the docker.
